qiskit/test/mock/fake_openpulse_3q.py

```diff
@@ -66,7 +66,64 @@ def __init__(self):
                 [100, 1000], [1000, 100], [100, 1000],
                 [100, 1000], [1000, 100], [100, 1000],
                 [1000, 100], [100, 1000], [1000, 100]
-            ]
+            ],
+            channels={
+                'acquire0': {
+                    'type': 'acquire',
+                    'purpose': 'acquire',
+                    'operates': {'qubits': [0]}
+                },
+                'acquire1': {
+                    'type': 'acquire',
+                    'purpose': 'acquire',
+                    'operates': {'qubits': [1]}
+                },
+                'acquire2': {
+                    'type': 'acquire',
+                    'purpose': 'acquire',
+                    'operates': {'qubits': [2]}
+                },
+                'd0': {
+                    'type': 'drive'
+                    'purpose': 'drive',
+                    'operates': {'qubits': [0]}
+                },
+                'd1': {
+                    'type': 'drive',
+                    'purpose': 'drive',
+                    'operates': {'qubits': [1]}
+                },
+                'd2': {
+                    'type': 'drive',
+                    'purpose': 'drive',
+                    'operates': {'qubits': [2]}
+                },
+                'm0': {
+                    'type': 'measure',
+                    'purpose': 'measure',
+                    'operates': {'qubits': [0]}
+                },
+                'm1': {
+                    'type': 'measure',
+                    'purpose': 'measure',
+                    'operates': {'qubits': [1]}
+                },
+                'm2': {
+                    'type': 'measure',
+                    'purpose': 'measure',
+                    'operates': {'qubits': [2]}
+                },
+                'u0': {
+                    'type': 'control',
+                    'purpose': 'cross-resonance',
+                   'operates': {'qubits': [0, 1]}
+                },
+                'u1': {
```

Comment:
Since this is a 3q device, if there is no u-channel connecting qubit 2, this device would have an isolated qubit. An additional u channel should be added.

Reply:
Fixed in [713d070](https://github.com/Qiskit/qiskit-terra/pull/4085/commits/713d070a1697e78ea43280181662042bdbb5c55d)